pkg/cloudprovider/instancetypes.go

```diff
@@ -69,19 +69,12 @@ type InstanceTypeProvider struct {
 }
 
 func NewInstanceTypeProvider(ctx context.Context, sess *session.Session, ec2api ec2iface.EC2API, subnetProvider *subnet.Provider,
```

Comment:
```suggestion
func NewInstanceTypeProvider(sess *session.Session, ec2api ec2iface.EC2API, subnetProvider *subnet.Provider,
```

I don't think we need this anymore